### ISSUE TITLE
tablefunc extension not available?

### ISSUE THREAD

**Issue by jhogendorn** [Expaso/hassos-addon-timescaledb#23]
```
CREATE EXTENSION IF NOT EXISTS tablefunc;
```

```
Query 1 ERROR: ERROR:  could not open extension control file "/usr/share/postgresql14/extension/tablefunc.control": No such file or directory
```

tablefunc is in the supported list of built in extensions:
https://kb-managed.timescale.com/en/articles/2754991-supported-postgresql-extensions

Is the image missing a package or something perhaps?

**Comment by tdn131** [comment]
I have the same issue with the two plugins I need for Teslamate. These plugins used to be included in PostgreSQL 12, but aren't anymore I read as of version 13. I checked: they are also not included in version 14.
create extension earthdistance;
"/usr/share/postgresql14/extension/earthdistance.control": No such file or directory
CREATE EXTENSION cube;
"/usr/share/postgresql14/extension/cube.control": No such file or directory

@Expaso The addons cube,earthdistance and tablefunc are amongst others included in postgresql14-contrib (see https://centos.pkgs.org/7/postgresql-14-aarch64/postgresql14-contrib-14.3-1PGDG.rhel7.aarch64.rpm.html )

Would it be possible to add the postgresql14-contrib, so we can enable these plugins? Would really appreciate this, because then I can disable my other PostgreSQL server and migrate Teslamate to your TimescaleDB addon.

**Comment by Expaso** [comment]
Thank you guys!
I have pulled in the PR as I want to give the addon a new update.

I will close this issue once the beta-testing is done and I have released the new version.

**Comment by Expaso** [comment]
New version released! 
I'm closing this one!

For any follow up's, please open a new issue when needed.
